package.json

```diff
@@ -175,7 +175,7 @@
     "webpack-stats-plugin": "^0.3.2"
   },
   "engines": {
-    "node": ">= 18.0.0",
+    "node": ">= 20.19.1",
     "npm": ">= 8.6.0"
```

Comment:
btw, npm on 20.19.1 has npm 10.8.2, so you probably should also change to that value here.